torch/csrc/jit/ivalue.h

```diff
@@ -427,6 +442,16 @@ inline IValue::IValue(Shared<IntList> v)
 inline IValue::IValue(std::vector<int64_t> v)
 : IValue(IntList::create(std::move(v))) {}
 
+inline IValue::IValue(Shared<ConstantString> v)
+: tag(Tag::String), retainable(true) {
+  as_retainable = v.detach();
+}
+inline IValue::IValue(const char * v)
+: IValue(ConstantString::create(std::move(v))) {}
+inline IValue::IValue(std::string v)
+: IValue(ConstantString::create(std::move(v.c_str()))) {}
```

Comment:
This is super unsafe!! The `const char *` data is managed by `v`, so it will be deallocated once the constructor finishes running! We really should capture an `std::string` (possibly a `const std::string`)

Reply:
Will change!! Thanks for the catch!